acceptancetests/assess_network_spaces.py

```diff
@@ -491,6 +486,9 @@ def main(argv=None):
     configure_logging(args.verbose)
 
     bs_manager = BootstrapManager.from_args(args)
+    # The bs_manager.client env's region doesn't normally get updated
+    # until we've bootstrapped. Let's force an early update.
+    bs_manager.client.env.update_config({'region': bs_manager.region})
```

Comment:
This could be (it's what update_config ends up doing anyway):
```python
bs_manager.client.env.
```

Not a blocker as this still works and is clear enough.

Reply:
I agree, .set_region would be more concise and clearer. Updating...